app/models/talentlms/facade.rb

```diff
@@ -61,19 +61,58 @@ def enroll(user, course_id)
       raise e unless e.message.include?('already enrolled')
     end
 
+    # Reset progress for a user in a course in TalentLMS
+    #
+    # @param user [User] the user
+    # @param course_id [Integer] the id of the course
+    def reset_user_progress(user, course_id = @api.courseid)
+      login = Login.find_by(user: user)
+      return false if login.nil?
+
+      @api.post('resetuserprogress', { course_id: course_id, user_id: login.lms_user_id })
+    end
+
+    # Log a completed training with the current TalentLMS course
+    #
+    # @param user [User] the user
+    # @param completion_date [Date] the date which the training was completed
+    # @param course_id [Integer] the id of the course
+    # @return [CompletedTraining] the completed training data
+    def log_course_completion(user, completion_date, course_id = @api.courseid)
+      login = Login.find_by(user: user)
+      return nil if login.nil?
+
+      return CompletedTraining.where(login: login, config: @api, course_id: course_id, completion_date: completion_date).first_or_create
```

Comment:
Can't believe rubocop didn't complain about this, but `return` is unnecessary.
```suggestion
      CompletedTraining.where(login: login, config: @api, course_id: course_id, completion_date: completion_date).first_or_create
```

Reply:
the return here has been removed